cernopendata/modules/fixtures/data/docs/cms-getting-started-2010/cms-getting-started-2010.md

```diff
@@ -20,6 +21,11 @@ $ cd CMSSW_4_2_8/src/
 $ cmsenv
 ```
 
+## <a name="container">"Do I have to use a virtual machine?"</a>
+
+Not necessarily. You can also try to analyze CMS data in a Docker container instead of in a virtual machine. If you are interested in that,
```

Comment:
```suggestion
Not necessarily. You can also try to analyse CMS data in a Docker container instead of in a virtual machine. If you are interested in that,
```